src/components/ImageUploader/README.md

```diff
@@ -7,8 +7,8 @@ Allows JPEG, PNG, and GIF file formats. HEIC is converted to JPEG by iOS with th
 ## Basic Usage With Upload Handler
 The upload hander (passed in as the `uploadHandlerFn` prop) should accept an object as the first argument. This object will contain the fields:
 
-- `image`: A `File` instance for the image to upload
-- `uploadProgressHandler`: A `Function` to push progress updates to the uploader, takes a number as an argument.
+- `imageFile`: A `File` instance for the image to upload
+- `uploadProgress`: A `Function` to push progress updates to the uploader, takes a number as an argument.
```

Comment:
also huh? there's no `uploadProgress` property, but there is a `setImageProgress` property: https://github.com/square/maker/blob/image-uploader-drag-n-drop/src/components/ImageUploader/src/ImageUploader.vue#L203

please update the docs and/or code so they match

also, changing the property names since the last release is a breaking change and requires a library major version bump, if you intend to stick with these name changes please communicate that these changes are breaking by:
1) renaming the PR title from `feat(image-uploader): add drag/drop to image picker` to `feat(image-uploader)!: add drag/drop to image picker`. please notice the added `!` which communicates to Maker to make a new major version release when the PR is merged in.
2) add a **Migration Guide from 6.x to 7.x** section to your PR description which gives people instructions on how to migrate their code to be compatible with the new major release, if you want some examples of prior written migration guides you can look at these PRs:
- https://github.com/square/maker/pull/148
- https://github.com/square/maker/pull/35

Reply:
The current version has the proper names, just the documentation was not updated. I don't think we need make it a breaking change.